AzureCalling/app/src/main/java/com/azure/samples/communication/calling/activities/SetupActivity.java

```diff
@@ -85,7 +88,8 @@ protected void onCreate(final Bundle savedInstanceState) {
         final CompletableFuture<Void> setupCompletableFuture = callingContext.setupAsync();
 
         final Intent intent = getIntent();
-        groupId = intent.getStringExtra(Constants.GROUP_ID);
+        callType = (JoinCallType) intent.getSerializableExtra(Constants.CALL_TYPE);
+        joinId = intent.getStringExtra(Constants.JOIN_ID);
```

Comment:
We should consider applying these renaming changes in main branch first to minimize changes later on when merging.